Sources/Tracing/Span.swift

```diff
@@ -583,61 +583,29 @@ extension SpanAttributes: ExpressibleByDictionaryLiteral {
 // ==== ----------------------------------------------------------------------------------------------------------------
 // MARK: Span Status
 
-/// Represents the status of a finished Span. It's composed of a canonical code in conjunction with an optional descriptive message.
+/// Represents the status of a finished Span. It's composed of a status code in conjunction with an optional descriptive message.
 public struct SpanStatus {
-    public let canonicalCode: CanonicalCode
+    public let code: Code
```

Comment:
Should it be possible to set one with undefined status code?

Reply:
> Should it be possible to set one with undefined status code?
> @ktoso 

Good question ... I don't think so, as unset basically means you haven't altered the status. The only use-case would be to set it at first, then reset it afterwards. But I don't think that's a real concern.